### PR TITLE
feat: add python3.11 support

### PR DESCRIPTION
**Pull request by moelasmar**
Adding the new python3.11 templates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM public.ecr.aws/lambda/python:3.11-preview
```
Comment (mndeveci) [inline python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile:1]: Is this still in preview or can we remove it?

**Review by mndeveci — CHANGES_REQUESTED** [review]

File: python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile (hunk 1 above)
Comment (moelasmar) [reply python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile:1]: good catch. I updated it in the build images and missed it here.

Comment (moelasmar) [reply python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile]: resolved

**Review by mndeveci — APPROVED** [review]

**Comment by mndeveci** [comment]
We might want to disable linting in the tests since it is causing failures for now

**Pull request merged**